site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -237,6 +238,78 @@ scenarios:
   ClusterQueue `team-b-cq` before admitting any new Workloads in `team-a-cq`.
   Therefore, Kueue ensures the `nominalQuota` quota for `team-b-cq` is met.
 
+You can test borrowing limit by create the following two jobs and local queues:
+
+```
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: LocalQueue
+metadata:
+  namespace: "default"
+  name: "user-queue-1"
```

Comment:
In the example local-queue in `config/samples` it targets `cluster-queue-1` and `cluster-queue-2` here it seem to target `team-a-cq`. Why do we need the sample/borrowingLimitSamples?

Reply:
Got it. I will remove files in config/samples/borrowingLimitSamples